chain/rosetta-rpc/src/adapters/mod.rs

```diff
@@ -352,6 +352,36 @@ impl From<NearActions> for Vec<crate::models::Operation> {
                     );
                 }
 
+                #[cfg(feature = "protocol_feature_nonrefundable_transfer_nep491")]
+                // Note(jakmeier): Both refundable and non-refundable transfers are considered as available balance.
```

Comment:
Added a TODO comment to consider this before stabilizing.

Reply:
I think this is a question that we should ask Coinbase since they own the spec. They usually are okay with making changes as long as they confirm from their side. @gmilescu could you help me with this, I am a bit out of loop with Coinbase.

Reply:
This change should be fine the way Rosetta handles this scenario. Thanks for checking.

Reply:
@nasmithan Could you take a look at this [change](https://github.com/near/nearcore/pull/10701/files#diff-843d19e05fcfdaab6b08c4cf1b67db9bf07a6805d60f2e2a9fdec45d85ff8a65L355-L382)?
In [discussion](https://github.com/near/NEPs/pull/491#issuecomment-1950280696) we decided to burn the nonrefundable balance instead of showing it as balance on the receiving account.

Reply:
Yup that should be fine